.github/pull_request_template.md

```diff
@@ -1,47 +1,43 @@
 Resolves #ISSUE(s).
-<!--- "Resolves #ISSUE" will automatically close #ISSUE when the PR is merged. However, if this PR won't 100% address the issue, you can use one of these patterns instead to simply link to the issue and not auto-close it:
-
-Partially addresses:
-- ISSUE(s)
-
-Addresses sub-task (n) in:
-- ISSUE(s)
+<!--- "Resolves #ISSUE" will automatically close #ISSUE when the PR is merged. However, if this PR won't 100% address 
+the issue and you don't want it to auto-close, you can write one of the following instead of "Resolves"
+- Partially addresses
+- Addresses sub-task (n) in
 -->
 
-
-## Pre-merge checklist
-
-<!--- Docs: A common case for documentation is the addition of new `make` goals. Descriptions should be documented for new goals both in the (i) `help` command at the bottom of the `mondo-ingest.Makefile` and (ii) `docs/developer/workflows.md`. -->
-
-### PR Overview
-<!--- Short paragraph or short bulleted list describing changes in this PR -->
-
+## Overview
+<!--- A few sentences describing changes / what problem is solved. Should describe the expected result of the changes. 
```

Comment:
## Overview section
Just making a few changes that didn't get merged from original PR (https://github.com/monarch-initiative/mondo-ingest/pull/430#discussion_r1482031772):
- Put _Overview_ above the _Pre-merge checklist_
- Updated instructions comment

I also renamed _PR Overview_ to _Overview_